src/app/admin-panel/page.tsx

```diff
@@ -4,21 +4,15 @@ import { getClassroomsSchema } from "@/schemas/classroomSchemas";
 import AdminPanelList from "./_components/AdminPanelList";
 import { getReservationsSchema } from "@/schemas/reservationSchemas";
 import { toTimestamp } from "@/utils/DateUtils";
+import { getExtraRoomsSchema } from "@/schemas/extraRoomSchemas";
+import { HOST } from "@/server-endpoints/host";
 
 export default async function AdminPanel() {
-  const equipments = await fetchGet(
-    "http://localhost:8080/equipments",
-    getEquipmentsSchema,
-  );
-  const classrooms = await fetchGet(
-    "http://localhost:8080/classrooms",
-    getClassroomsSchema,
-  );
+  const equipments = await fetchGet(`${HOST}/equipments`, getEquipmentsSchema);
+  const classrooms = await fetchGet(`${HOST}/classrooms`, getClassroomsSchema);
+  const extraRooms = await fetchGet(`${HOST}/extra-rooms`, getExtraRoomsSchema);
```

Comment:
Those 3 request seem to not rely on each other so are we waiting for each one to finish before starting the next one? We should use Promise.all here